src/Agent.Plugins/PipelineArtifact/FileContainerProvider.cs

```diff
@@ -37,13 +38,15 @@ public FileContainerProvider(VssConnection connection, IAppTraceSource tracer)
             }
         }
 
-        public async Task DownloadSingleArtifactAsync(PipelineArtifactDownloadParameters downloadParameters, BuildArtifact buildArtifact, CancellationToken cancellationToken)
+        public async Task DownloadSingleArtifactAsync(PipelineArtifactDownloadParameters downloadParameters, BuildArtifact buildArtifact, CancellationToken cancellationToken, AgentTaskPluginExecutionContext context)
         {
+            context.Warning($"Please use Download Build Artifact task for downloading 'Build Artifact' type artifact");
```

Comment:
should this be localized? Also, should we include a link to more documentation on what we want the users to do?

Reply:
Yeah, added localization and link.